From e93dbd66973040f1e0afcba0dc7c712c27d75d59 Mon Sep 17 00:00:00 2001
From: Matthias Clasen <mclasen@redhat.com>
Date: Mon, 1 May 2023 23:27:52 -0400
Subject: [PATCH] Drop -Werror=array-bounds

gcc has strange issues with this and produces false
positives that recently started breaking the build of
pango as a subproject in gtk.

See e.g. https://gcc.gnu.org/bugzilla/show_bug.cgi?id=105523
---
 meson.build | 2 --
 1 file changed, 2 deletions(-)

--- pango-1.50.11/meson.build
+++ pango-1.50.11/meson.build
@@ -88,7 +88,6 @@ '-Wundef',
     '-Wuninitialized',
     '-Wunused',
     '-Werror=address',
-    '-Werror=array-bounds',
     '-Werror=empty-body',
     '-Werror=implicit',
     '-Werror=implicit-fallthrough',
@@ -131,7 +130,6 @@ '-Werror=missing-braces',
     '-Werror=sequence-point',
     '-Werror=return-type',
     '-Werror=trigraphs',
-    '-Werror=array-bounds',
     '-Werror=write-strings',
     '-Werror=address',
     '-Werror=int-to-pointer-cast',
